### ISSUE TITLE
Fix the CI

### ISSUE THREAD

**Issue by vchrombie** [nyuoss/glaceon-google-suite-tasks#24]
The CI fails, because of the tests. Which is in turn, probably because of the fault in the importing the source files.

<img width="607" alt="image" src="https://github.com/nyuoss/glaceon-google-suite-tasks/assets/25265451/4eec833d-6ff0-4434-ae9a-f1efe92489f3">

This should fix it I guess.

_Originally posted by @vchrombie in https://github.com/nyuoss/glaceon-google-suite-tasks/issues/23#issuecomment-2109243089_

**Comment by Yanglin-Tao** [comment]
Thanks for the suggestion. The import issue has been fixed in this PR, https://github.com/nyuoss/glaceon-google-suite-tasks/pull/25, however the CI is still failing and likely due to the setup of mocked Google APIs.

**Comment by vchrombie** [comment]
> Thanks for the suggestion. The import issue has been fixed in this PR, #25, however the CI is still failing and likely due to the setup of mocked Google APIs.

You are right. Instead of setting the mocked Google APIs, you can use mocking in pytest.
